docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -1074,6 +1074,46 @@ File containing the node's private key.
 List of comma-separated [multiaddresses](https://docs.libp2p.io/reference/glossary/#multiaddr)
 of static peers.
 
+### rest-api-cors-origins
+
+=== "Syntax"
+
+    ```bash
+    --rest-api-cors-origins[=<url>[,<url>...]...] or "all" or "*"
+    ```
+
+=== "Command Line"
+
+    ```bash
+    --rest-api-cors-origins="http://medomain.com","https://meotherdomain.com"
+    ```
+
+=== "Environment Variable"
+
+    ```bash
+    TEKU_REST_API_CORS_ORIGINS="http://medomain.com","https://meotherdomain.com"
+    ```
+
+=== "Configuration File"
+
+    ```bash
+    rest-api-cors-origins: ["http://medomain.com","https://meotherdomain.com"]
+    ```
+
+A list of domain URLs for CORS validation. You must enclose the URLs in double quotes and separate
+them with commas.
+
+Listed domains can access the node using HTTP REST API calls. If your client interacts with Teku
+using a browser app (such as a block explorer), add the client domain to the list.
+
+The default value is "none." If you don't list any domains, browser apps can't interact with your
+Teku node.
+
+!!! tip
+
+    For testing and development purposes, use `all` or `*` to accept requests from any domain.
```

Comment:
`*` is supported but `all` is not supported for the moment. Can you ask if we should ? If yes i will add suport to `all` value as well.

Reply:
Ideally yes, having `all` supported would be nice, but I don't think it's a priority as anyway the `all` or `*` setting is only recommended for dev and not prod.
I think updating this PR is going to be more pragmatic ;)